tests/samples/expected/ocaml.ml

```diff
@@ -17,7 +17,7 @@
 
 type t = {
   mutable buffer: bytes;
-  mutable position: int; (* End-of-line comment *)
+  mutable position: int (* End-of-line comment *);
```

Comment:
It's rather unusual. But why not. (I guess that with semi-column-terminated line, I would put the comment before the line anyway). Register this as: no opinion.

Out of curiosity, what is it a solution to?

Reply:
Once again, not sure about what your question is:
  1. Why do we add a semicolon if there is none?
    It was like this before I got involved in the project(I think Tor implemented it), so I cannot provide details about the history, but I can say that I find it quite logical to do so, arguments are given in the first message of the PR.
  2. Why does it have to change?
    The current implementation, was simply "add a semicolon after a field_delimiter unless there is already one", the issue with this was that when there is a field_delimiter, then a comment and then a semicolon `x : float (* The x-axis coordinate *);` , a semicolon was added after the field_delimiter `x : float; (* The x-axis coordinate *);` and this was invalid Ocaml because there were 2 semicolons for only one field. Hence a decision had to be taken about where does this semicolon go. Since I discovered this issue while formatting this file https://github.com/Deducteam/lambdapi/blob/2933ac40c75a53b773b421a7069741c5aa07c0a8/src/core/sign.ml#L9
I decided that it was a good idea to put it after the comment. But I do not really care.

Reply:
The question was indeed about (2).

Anyway, it's fine how it is, feel free to merge.

Reply:
Is it easy to put the semicolon before the comment? The two of you may not feel too strongly either way, but I would sleep better at night 🙂

Just comparing it to other languages with other comment options, this would not work:

```
mut position: i64 // End-of-line comment;
```

Reply:
I made the change you requested @torhovland